### DISCUSSION TITLE
How using runtimeCaching with SSR, no example or clear doc

### DISCUSSION THREAD

**Discussion by romain130492** [quasarframework/quasar#13590] [CLI - SSR mode]
Hey
Just wondering how we are supposed to use runtimeCaching  with SSR, (Quasar v1)
I can see in the doc that we add an object ` componentCache`, but what else? Is there any repo or example using that set-up

```js
 ssr: {
   
    componentCache: {...}
}

```

Thanks!

**Comment by suleimanas** [comment]
https://developer.chrome.com/docs/workbox/reference/workbox-build/#type-RuntimeCaching

**Reply by romain130492** [reply]
but that has nothing to do with the implementation on quasar isn't?

**Reply by suleimanas** [reply]
:) runtimeCaching basically you would use it if you want to extend your serviceworker caching strategy, buy why bother. workbox doing great job already
